chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -108,29 +108,28 @@ export enum TelemetryEvent {
     EndChatFailed = "EndChatFailed",
     SettingCustomContext = "SettingCustomContext",
     WebChatLoaded = "WebChatLoaded",
-    LCWChatButtonClicked = "LCWChatButtonClicked",
+    LCWChatButtonActionCompleted = "LCWChatButtonActionCompleted",
+    LCWChatButtonActionStarted = "LCWChatButtonActionStarted",
     LCWChatButtonShow = "LCWChatButtonShow",
 
-    WidgetLoadStarted = "WidgetLoadStarted",
-    WidgetLoadComplete = "WidgetLoadComplete",
-    WidgetLoadFailed = "WidgetLoadFailed",
+    WidgetStartChatStarted = "WidgetStartChatStarted",
+    WidgetStartChatCompleted = "WidgetStartChatCompleted",
+    WidgetStartChatFailed = "WidgetStartChatFailed",
     StartChatMethodException = "StartChatMethodException",
     CloseChatCall = "CloseChatCall",
     CloseChatMethodException = "CloseChatMethodException",
     PrechatSurveyLoaded = "PrechatSurveyLoaded",
     PrechatSurveyExpected = "PrechatSurveyExpected",
-    PrechatSubmitted = "PrechatSubmitted",
+    PrechatSubmitCompleted = "PrechatSubmitCompleted",
     StartChatSDKCall = "StartChatCall",
-    StartChatEventRecevied = "StartChatEventReceived",
+    StartChatEventReceived = "StartChatEventReceivedCompleted",
     EndChatSDKCall = "EndChatSDKCall",
     PrepareEndChat = "PrepareEndChat",
-    EndChatEventReceived = "EndChatEventReceived",
+    EndChatEventReceived = "EndChatEventReceivedCompleted",
     WindowClosed = "WindowClosed",
     OnNewMessageFailed = "OnNewMessageFailed",
     OnNewMessageAudioNotificationFailed = "OnNewMessageAudioNotificationFailed",
     DownloadTranscriptResponseNullOrUndefined = "DownloadTranscriptResponseNullOrUndefined",
```

Comment:
have we identified adjustments needed in OOB? , like broadcast service listener ? , also this will break multiple Monitors and CAPSLA,

- please test using a tgz with this PR in OOB to identify if any functionality could be affected with this rename of events
please create a task to take care of all queries immediately after this PR is merged ,